src/custom/hooks/useRefetchPriceCallback.ts

```diff
@@ -56,6 +66,19 @@ export function useRefetchQuoteCallback() {
         // Get the quote
         const [price, fee] = await getQuote(params)
 
+        const previouslyUnsupportedToken =
+          gpUnsupportedTokens && (gpUnsupportedTokens[sellToken] || gpUnsupportedTokens[buyToken])
```

Comment:
Why not using `isUnssuportedToken`

Actually, why not making isUnssuportedToken able to receive a second optional token, so you can make it cleaner when you use it:
`isUnssuportedToken(sellToken, buyToken)`

Reply:
i opted not for the second param but did change this to the fn

Reply:
I didn't understand your answer. Anyways, my was a nitpick since we do that a lot, but we can skip it